docs/admin/runai-setup/maintenance/alert-monitoring.md

```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
            alertmanagerConfigSelector:
               matchLabels:
                  alertmanagerConfig: runai
-        EOF
 
-3. Exposing the Alertmanager Service
+2. Validate that your alertmanager instance has started:
 
-        cat <<EOF | kubectl apply -f - 
-        apiVersion: v1
-        kind: Service
-        metadata:
-           name: alertmanager-runai
-           namespace: runai
-        spec:
-           type: NodePort
-           ports:
-           - name: web
-              nodePort: 30903
-              port: 9093
-              protocol: TCP
-              targetPort: web
-           selector:
-              alertmanager: runai
-        EOF
+        kubectl get alertmanager -n runai
+
+3. Validate that prometheus operator has created a service for alertmanager:
+
+         kubectl get svc alertmanager-operated -n runai
 
 ## Configuring Prometheus to Send Alerts
 
 1. Edit the Prometheus configuration:
 
       `kubectl edit prometheus runai -n runai`
 
-2. Add the following to the `alerting` section:
+2. Add the following to the `spec.alerting` section:
 
         alerting:
            alertmanagers:
               - namespace: runai
-                name: alertmanager-runai
+                name: alertmanager-operated
                 port: web
 
 3. Save and exit the editor. The configuration will be automatically reloaded.
 
 ## Configuring Alertmanager for Custom Email Alerts
+In each step, copy the contents to a file and apply it to the cluster using `kubectl apply -f`
```

Comment:
If you are referring to code snippiets, then you need to specify that, and more specifically which ones need to be copied, to which file, and to where in the file. If this a new file then that needs to be stated, if it is an already existing file, then you need to be specific about the name of the file.

Reply:
added